Mastering HTML Forms/HTML Forms and Form Tags.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Creating HTML Forms</title>
+  </head>
+  <body>
+    <!-- two most use attribute of form tag are action and method. action is use to send form submit data to the define path/url, method is to identify the method to send data (there are two ways to send data: POST/GET, which we understand in detail in js tutorial)-->
+    <form action="submit-form.php" method="post">
+      <!-- Form elements go here -->
```

Comment:
```suggestion
      <!-- Form elements go here -->
      why form element does not have any child element? why we create Empty form element.
```

Reply:
@aarif135 this file will explain only **form tag** that's why only form tag is visible